src/components/Main/Modal/QRCodeModal/QRCodeModal.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <modal-frame title="QR Code" icon-mdi icon-name="qrcode">
+    <div
+      v-if="!err"
+      :class="$style.qrCodeModalImage"
+      :style="styles.container"
+    />
+    <modal-section v-else title="エラー" :description="err" />
+  </modal-frame>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent,
+  reactive,
+  onMounted,
+  onUnmounted,
+  toRefs,
+  Ref
+} from '@vue/composition-api'
+
+import { makeStyles } from '@/lib/styles'
+import ModalFrame from '../Common/ModalFrame.vue'
+import ModalSection from '../Common/ModalSection.vue'
+import apis from '@/lib/api'
+
+const useStyles = (url: Ref<string>) =>
+  reactive({
+    container: makeStyles(() => ({
+      backgroundImage: `url(${url.value})`
+    }))
+  })
+
+const useQrCode = () => {
+  const state = reactive({
+    url: '',
+    err: ''
+  })
+
+  onMounted(() => {
+    makeBlob()
+      .then(value => {
+        state.url = value
+      })
+      .catch(err => {
+        state.err = err
+      })
```

Comment:
```suggestion
  onMounted(async () => {
    try {
      state.url = await makeBlob()
    } catch (err) {
      state.err = err
    }
```
好みだけどasync awaitで書きたい